Bottom: 1f0d7723fc3e565b562fbe373a2a801c85f2278e
Top:    4a2a615817b3d249d54a9821bb401af6f136f16d
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 15:58:48 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/direct.c b/daemon/direct.c
index 38c8543..5c1006d 100644
--- a/daemon/direct.c
+++ b/daemon/direct.c
@@ -31,7 +31,6 @@
 #include <sys/stat.h>
 #include <sys/time.h>
 #include <sys/poll.h>
-#include <sys/mount.h>
 #include <sys/vfs.h>
 #include <sched.h>
 
diff --git a/daemon/indirect.c b/daemon/indirect.c
index 45758ef..b2b4bce 100644
--- a/daemon/indirect.c
+++ b/daemon/indirect.c
@@ -30,7 +30,6 @@
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <sys/time.h>
-#include <sys/mount.h>
 #include <sys/vfs.h>
 #include <sched.h>
 
diff --git a/daemon/spawn.c b/daemon/spawn.c
index 3b4a009..0f7d56a 100644
--- a/daemon/spawn.c
+++ b/daemon/spawn.c
@@ -24,7 +24,6 @@
 #include <poll.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
-#include <sys/mount.h>
 
 #include "automount.h"
 
diff --git a/lib/mounts.c b/lib/mounts.c
index 6315201..d7b1fe7 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -18,7 +18,6 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/ioctl.h>
-#include <sys/mount.h>
 #include <sys/wait.h>
 #include <ctype.h>
 #include <stdio.h>
diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index ae1caf7..d75e117 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -29,7 +29,6 @@
 #include <sys/vfs.h>
 #include <sys/utsname.h>
 #include <netinet/in.h>
-#include <sys/mount.h>
 #include <linux/fs.h>
 
 #define MODULE_PARSE
